flink-runtime/src/main/java/org/apache/flink/runtime/state/MultiStreamStateHandle.java

```diff
@@ -1,104 +0,0 @@
-/*
```

Comment:
That was a dead code, right? Please extract removal of this (and `MultiStreamStateHandleTest`) to separate commit.

Reply:
Yes, they are deleted in later commits anyways.